src/plugins/irc-events/topic.js

```diff
@@ -9,27 +9,24 @@ module.exports = function(irc, network) {
 			return;
 		}
 		var from = data.nick || chan.name;
-		var self = false;
-		if (from.toLowerCase() == irc.me.toLowerCase()) {
-			self = true;
-		}
 		var topic = data.topic;
+
 		var msg = new Msg({
 			type: Msg.Type.TOPIC,
 			mode: chan.getMode(from),
 			from: from,
 			text: topic,
-			self: self
+		        self: (from.toLowerCase() === irc.me.toLowerCase())
```

Comment:
Argh, I missed this part, sorry! Can you replace the 4 spaces with a tab, please? :-(